data/meson.build

```diff
@@ -2,32 +2,42 @@
 # install the icons
 install_data(
     'icons/hicolor/scalable/apps/@0@.svg'.format(meson.project_name()),
-    install_dir: join_paths(get_option('datadir'), 'icons/hicolor/scalable/apps')
+    install_dir: get_option('datadir') / 'icons' / 'hicolor' / 'scalable' / 'apps'
 )
 install_data(
     'icons/hicolor/scalable/apps/@0@-symbolic.svg'.format(meson.project_name()),
-    install_dir: join_paths(get_option('datadir'), 'icons/hicolor/scalable/apps')
+    install_dir: get_option('datadir') / 'icons' / 'hicolor' / 'symbolic' / 'apps'
```

Comment:
Instead of `get_option('datadir') everywhere, why not declare `datadir =  get_option('datadir')` and then `pkgdatadir = join_paths(datadir, 'dir_name')` for common directories like datadir/tilix?

Reply:
Thanks for the review Chris! Will fix that soon